docker-compose.yml

```diff
@@ -1,14 +1,20 @@
 version: '3.1'
 
 services:
+  jaeger:
+    image: jaegertracing/opentelemetry-all-in-one:latest
+    ports:
+      - 16686:16686
+      - 13133:13133
+      - 55680:55680
   ads:
     image: wso2inc/ads-service:v0.1.0
   cart:
     image: wso2inc/cart-service:v0.1.0 #add redis
   checkout:
     image: wso2inc/checkout-service:v0.1.0
     environment:
-      BAL_CONFIG_DATA: catalogHost="catalog" \n cartHost="cart" \n currencyHost="currency" \n shippingHost="shipping" \n paymentHost="payment" \n emailHost="email"
+      BAL_CONFIG_DATA: catalogHost="catalog" \n cartHost="cart" \n currencyHost="currency" \n shippingHost="shipping" \n paymentHost="payment" \n emailHost="email" catalogTimeout=3.0 \n cartTimeout=3.0 \n currencyTimeout=3.0 \n shippingTimeout=3.0 \n paymentTimeout=3.0 \n emailTimeout=3.0
```

Comment:
Can't we have the timeout 3.0 as the default value in configurables? Then we can reduce the lines here. If this BAL_CONFIG_DATA increases, the readability heavily gets reduce :( wdyt?

Reply:
ack
will change

Reply:
Added https://github.com/ballerina-guides/gcp-microservices-demo/pull/37/commits/e37e5cac02f8e9d418281a152d86b8c3a120eaf9